### PR TITLE
Update links to internal pages

### PR DESCRIPTION
**Pull request by omeneses**
1. Link to Trello's task: https://trello.com/c/C0tEpSKj/37-general-actualizar-links-1
2. task's design in zeplin: Does not apply
3. Component's screenshot
Screen Shot 2020-08-23 at 10 30 54 PM

### CODE REVIEW

File: src/data/data.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const pageData = {
           'Promovemos la recolección del aceite usado de cocina, para que pueda ser utilizado nuevamente en una energía renovable.',
         button: {
           text: 'Registrate',
-          href: 'Register',
+          href: '/register.html',
```
Comment (jsebax) [inline src/data/data.js]: In the navigation links you have `/registro.html`, but here you have `/register.html`. These two links need to match. Please update.

Comment (omeneses) [reply src/data/data.js]: Done!!!

**Review by ArleyAV — APPROVED** [review]
Well done Omar

File: src/data/data.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const pageData = {
           'Promovemos la recolección del aceite usado de cocina, para que pueda ser utilizado nuevamente en una energía renovable.',
         button: {
           text: 'Registrate',
-          href: 'Register',
+          href: '/registro.html',
```
Comment (jsebax) [inline src/data/data.js]: Sorry @omeneses, could you please remove the extension `.html` from the links. It is not necessary anymore

Comment (omeneses) [reply src/data/data.js]: Done!!!

**Review by jsebax — APPROVED** [review]

**Pull request merged**